### ISSUE TITLE
Highlightning "if" in the name

### ISSUE THREAD

**Issue by includealex** [EOLangVSCode/eo-vscode#10]
At this moment names, that contain `if` part, are highlighted. For instance, this code
```eo
[] > check-if-true
  assert-that > @
    TRUE
    $.equal-to
      TRUE
```
will look like this:
![image](https://user-images.githubusercontent.com/70340829/198244390-095fa1ed-e839-4ecb-ae0c-8d6f831ecf12.png)

**Comment by EduardoFernandes1410** [comment]
Thanks for the feedback! This should be fixed soon when we roll out semantic highlighting. Stay tuned.

**Comment by EduardoFernandes1410** [comment]
Hey there! This issue was fixed in the new 0.1.0 version of the extension. Thanks again for providing feedback!
